src/components/__tests__/RouteItemDisplay.cy.js

```diff
@@ -142,20 +156,124 @@ function coreTests() {
       .should('be.visible')
       .invoke('width')
       .should('be.eq', 18);
+    // avatar transport
+    cy.dataCy('avatar-transport').should('be.visible');
+    cy.dataCy('avatar-transport').invoke('height').should('be.eq', 32);
+    cy.dataCy('avatar-transport').invoke('width').should('be.eq', 32);
     // icon transport
     cy.dataCy('icon-transport')
       .should('be.visible')
-      .invoke('height')
-      .should('be.eq', 24);
-    cy.dataCy('icon-transport')
-      .should('be.visible')
-      .invoke('width')
-      .should('be.eq', 24);
+      .and('have.css', 'font-size', '18px');
+    cy.dataCy('icon-transport').invoke('height').should('be.eq', 18);
```

Comment:
Repeatable string or number should be const.

Reply:
Extracted to a variable.